assets/js/blocks/featured-category/index.js

```diff
@@ -37,6 +39,21 @@ registerBlockType( 'woocommerce/featured-category', {
 	supports: {
 		align: [ 'wide', 'full' ],
 		html: false,
+		color: {
+			text: true,
+			background: true,
+		},
```

Comment:
Background and text color are enabled by default so this can be shortened like this:
```suggestion
		color: true,
```